lib/src/components/SimpleComponents.tsx

```diff
@@ -63,6 +63,7 @@ export const NxList = Object.assign(withClass('ul', 'nx-list'), {
 });
 
 export const NxPageTitle = Object.assign(withClass('div', 'nx-page-title'), {
+  Subtitle: withClass('h2', 'nx-page-title__sub-title'),
```

Comment:
There are unit tests for these

Reply:
> There are unit tests for these

I did not know that: https://github.com/sonatype/sonatype-react-shared-components/pull/363/commits/b5db6af974b1b5654cc5577d5b65d86f02a5cf50